velox/functions/sparksql/String.h

```diff
@@ -845,4 +845,103 @@ struct TranslateFunction {
   }
 };
 
+/// Decodes `string` in 'application/x-www-form-urlencoded' format using UTF-8
+/// encoding scheme.
+template <typename T>
+struct UrlDecoderFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  // ASCII input does NOT always produce ASCII result.
+  static constexpr bool is_default_ascii_behavior = false;
+
+  int decode(StringView input) {
+    assert(input.size() == 2);
```

Comment:
VELOX_USER_CHECK

Reply:
Thanks for your comment! It looks not quite necessary to keep this check. So I just removed this statement. Thanks!